server/config.go

```diff
@@ -62,7 +64,7 @@ type Config struct {
 	LeaderLease int64 `toml:"lease" json:"lease"`
 
 	// Log related config.
-	Log logutil.LogConfig `toml:"log" json:"log"`
+	Log log.Config `toml:"log" json:"log"`
```

Comment:
Is it guaranteed to be compatible with the original configuration file?

Reply:
Yes.